tests/test_feature.py

```diff
@@ -33,19 +34,42 @@ def features_extractor_sliding_window(signal, _, hop_size):
     return np.ones((NUM_CHANNELS, NUM_FEATURES, num_time_steps))
 
 
+def test_deprecated_unit_argument():
+    if (
+            audeer.LooseVersion(audinterface.__version__)
+            < audeer.LooseVersion('1.2.0')
+    ):
+        with pytest.warns(UserWarning, match='is deprecated'):
+            interface = audinterface.Feature(
+                ['a'],
+                win_dur=1000,
+                unit='samples',
+            )
+            assert interface.win_dur == '1000'
```

Comment:
This test fails as `kwargs` inside `audinterface.Feature` is `{}` and does not contain `unit`. Any idea why this could be the case?

Reply:
It's removed by the decorator. Maybe the `audeer.deprecated_keyword_argument` does not forward it?

Reply:
Yes, it's removed here: https://github.com/audeering/audeer/blob/9798923c7ef1f8dfdadce6e21c65678688fcd6aa/audeer/core/utils.py#L166

Reply:
So the question is if we should modify the behavior of `@audeer.deprecated_keyword_argument` and remove it only from `kwargs` when a new keyword arg is specified, or if I should simply use custom code here instead.

Reply:
If within the mapping function we had access to the other arguments, we could directly modify `win_dur` there. This would probably be the cleanest solution. But I think the workaround is fine for the moment.

Reply:
I created https://github.com/audeering/audeer/issues/69

Reply:
> is not a solution as it will suggest to the user she/he should use `unit` instead of `unit`.

Hehe, good that you noticed that :)